olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -907,9 +907,9 @@ spec:
                       - name: "IMAGE_eventing-controller__eventing-controller"
                         value: "registry.redhat.io/openshift-serverless-1/kn-eventing-controller-rhel8@sha256:1b4856760983e14f50028ab3d361bb6cd0120f0be6c76b586f2b42f5507c3f63"
                       - name: "IMAGE_eventing-istio-controller__eventing-istio-controller"
-                        value: "registry.redhat.io/openshift-serverless-1/kn-eventing-istio-controller-rhel8@sha256:695f843198103d1035c0a503bad9f00b37fa6e6dfe5f3919e927869ff536d93e"
+                        value: "registry.redhat.io/openshift-serverless-1/kn-eventing-istio-controller-rhel8@sha256:edf7352fda07e8223bdefb8e791dba5e54356a7c11df596156bc4ded67e1a338"
```

Comment:
/hold
this Konflux build failed https://github.com/openshift-knative/eventing-istio/runs/35657372004